src/main/java/io/xpring/xrpl/DefaultXpringClient.java

```diff
@@ -131,4 +130,22 @@ public RawTransactionStatus getRawTransactionStatus(String transactionHash) thro
 
         return new RawTransactionStatus(response);
     }
+
+    private XRPDropsAmount getMinimumFee() {
+        return this.getFee().getDrops().getMinimumFee();
```

Comment:
This is kind of a weird API.

`getFee().getDrops().getMinimumFee()`?

What is this coming from?

Reply:
`getFee()` is a method (line 138) which returns a `GetFeeResponse` protocol buffer. 

This protocol buffer accesses members through `get<Foo>` so the `getDrops()` returns a message with getMinimumFee().

This chaining is the same as:
```
GetFeeResponse gfr = this.getFee()
Fee drops = gfr.getDrops(); // this member is oddly named, its generated.
return drops.getMinimumFee().
```